src/proxy_conf.rs

```diff
@@ -0,0 +1,295 @@
+// TODO these constant destinations are not final.
+
+use std::env;
+use std::path::{Path, PathBuf};
+
+// Where the cache and socket are stored by default
+pub const RUN_DIR: &str = "/run/podman";
+
+// Default UDS path for gRPC to communicate on.
+pub const DEFAULT_UDS_PATH: &str = "/run/podman/nv-proxy.sock";
+// Default configuration directory.
+pub const DEFAULT_CONFIG_DIR: &str = "";
+// Default Network configuration path
+pub const DEFAULT_NETWORK_CONFIG: &str = "/dev/stdin";
+// Default epoll wait time before dhcp socket times out
+pub const DEFAULT_TIMEOUT: isize = 8;
+// Proxy server gRPC socket file name
+pub const PROXY_SOCK_NAME: &str = "nv-proxy.sock";
+// Where leases are stored on the filesystem
+pub const CACHE_FILE_NAME: &str = "nv-proxy.lease";
+
+/// Get the RUN_DIR where the proxy cache and socket
+/// are stored
+///
+///
+/// # Arguments
+///
+/// * `run_cli`:
+///
+/// returns: String
+///
+/// # Examples
+///
+/// ```
+///
+/// ```
+pub fn get_run_dir(run_cli: Option<&str>) -> String {
+    // if environment, return it
+    // if opt, return it
+    // return default
+
+    match env::var("RUN_DIR") {
```

Comment:
please do not use generic names, for env vars. They should be namespaced so we do not conflict with outer projects. Maybe `NETAVARK_RUN_DIR`?